dist/index.js

```diff
@@ -3566,10 +3566,17 @@ const core = __nccwpck_require__(186);
 const exec = __nccwpck_require__(514);
 const { promises: fs } = __nccwpck_require__(747);
 
-const host =
-  core.getInput("owner") === "Staging-ObelusFamily"
-    ? "https://engine-staging.wilco.gg"
-    : "https://engine.wilco.gg";
```

Comment:
Why dist is part of the PR?

Reply:
it automatically compile the code so we can test it

Reply:
why do we have dist in the repo? 🤔 should be added to `.gitignore`, no?

Reply:
ah oops this isn't wilco-engine